Code/GraphMol/MolDraw2D/MolDraw2DCairo.cpp

```diff
@@ -99,6 +99,19 @@ void MolDraw2DCairo::setColour(const DrawColour &col) {
   cairo_set_source_rgba(dp_cr, col.r, col.g, col.b, col.a);
 }
 
+namespace {
+void setDashes(cairo_t *dp_cr, const DashPattern &dashes) {
+  if (dashes.size()) {
+    auto *dd = new double[dashes.size()];
```

Comment:
Any reason this can't just be

```
cairo_set_dash(dp_cr, dashes.data(), dashes.size(), 0);
```

Reply:
I believe DashPattern is a std::vector

Reply:
This would remove the if(dashes.size) requirement, since dashes.data() returns nullptr when size == 0